src/app.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
```

Comment:
I don't think the `Component` import is used. You just access `React.Component` everywhere which is fine.